packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -23,35 +23,170 @@ import fs from "fs";
 
 import { duration } from "../../../tasit-sdk/dist/testHelpers/helpers";
 
-const { ONE, TEN, ONE_HUNDRED, ONE_THOUSAND } = constants;
+const { ONE, TEN, ONE_HUNDRED, ONE_THOUSAND, BILLION, WeiPerEther } = constants;
 
-const createMultipleParcels = async (
-  landContract,
-  parcels,
-  beneficiaryAddress,
-  contractOwnerWallet
-) => {
-  let xArray = [];
-  let yArray = [];
+let network = process.env.NETWORK;
```

Comment:
Worth documenting somewhere that someone would need to set this? Or logging if it's not set?

Reply:
Done.

Reply:
Thanks! Resolving